packages/convergence/src/converge-on.js

```diff
@@ -23,7 +23,7 @@
  * @returns {Promise} resolves if the assertion passes at least once;
  * if `always` is true, then rejects at the first error instead
  */
-export default function convergeOn(assertion, timeout = 2000, always, useStats) {
+export default function convergeOn (assertion, timeout = 2000, always, useStats) {
```

Comment:
I don't know how I feel about this rule. Seems like not the way most setups work?

Reply:
I will change that.